llvm/include/llvm/BinaryFormat/XCOFF.h

```diff
@@ -334,9 +334,20 @@ enum CFileLangId : uint8_t {
 };
 
 enum CFileCpuId : uint8_t {
-  TCPU_PPC64 = 2, ///< PowerPC common architecture 64-bit mode.
-  TCPU_COM = 3,   ///< POWER and PowerPC architecture common.
-  TCPU_970 = 19   ///< PPC970 - PowerPC 64-bit architecture.
+  TCPU_INVALID = 0, ///< Invalid id - assumes POWER for old objects.
```

Comment:
Is there any document where do these constants come from?

Reply:
Yes, these values are found in the file `/usr/include/aouthdr.h` on AIX.